### PR TITLE
mpvc: init at unstable-2017-03-18

### PR DESCRIPTION
**Pull request by neeasade**
###### Motivation for this change
package mpvc.

FYI this was first time using wrapper for me.

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/mpvc/default.nix [hunk 1]
```diff
@@ -10,23 +10,14 @@ stdenv.mkDerivation rec {
...
-    make PREFIX=$out
-  '';
+  makeFlags = [ "PREFIX=$(out)" ];
+  installFlags = [ "PREFIX=$(out)" ];
 
   postInstall = ''
-    makeWrapper ${socat}/bin/socat $out/bin/socat \
-      --prefix PATH : "$out/bin"
-    wrapProgram $out/bin/mpvc
+    wrapProgram $out/bin/mpvc --prefix PATH : "${socat}/bin/"
```
Comment (Mic92) [inline pkgs/applications/misc/mpvc/default.nix]: I simplied the build process a little bit.

File: pkgs/applications/misc/mpvc/default.nix [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  '';
+
+  buildInputs = [ socat makeWrapper ];
+  outputs = [ "out" ];
+
+  meta = {
+    description = "A mpc-like control interface for mpv";
+    homepage = https://github.com/wildefyr/mpvc;
+    license = stdenv.lib.licenses.mit;
+    platforms = stdenv.lib.platforms.linux;
```
Comment (Mic92) [inline pkgs/applications/misc/mpvc/default.nix]: Do you want to maintain this package?

File: pkgs/applications/misc/mpvc/default.nix (hunk 1 above)
Comment (neeasade) [inline pkgs/applications/misc/mpvc/default.nix]: thanks

File: pkgs/applications/misc/mpvc/default.nix (hunk 2 above)
Comment (neeasade) [inline pkgs/applications/misc/mpvc/default.nix]: sure yes -- should I edit this PR to add to maintainers list?

Comment (Mic92) [inline pkgs/applications/misc/mpvc/default.nix]: yes

**Comment by Mic92** [comment]
Thanks!

**Pull request merged**